website/blog/2020-03-24-version-0.62.md

```diff
@@ -0,0 +1,86 @@
+
+Flipper provides the following features out of the box:
+
+- Network Inspector
+- Logout Inspector
+- Crash Reporter
+- Device logs
+
+There is also a marketplace of custom Flipper plugins you can install:
+
+[TODO: Flipper Image]
+
+For more information, check out the Flipper documentation here.
+
+## Better dark mode support
+
+We’ve added a new Appearance module to provide access to the user's appearance preferences, such as their preferred color scheme (light or dark).
+
+```
+const colorScheme = Appearance.getColorScheme();
+if (colorScheme === 'dark') {
+  // Use dark color scheme
+}
+```
+
+We’ve also added a hook to subscribe to state updates to the users preferences automatically:
+
+```
+import {Text, useColorScheme} from 'react-native';
+
+const MyComponent = () => {
+  const colorScheme = useColorScheme();
+  return <Text>useColorScheme(): {colorScheme}</Text>;
+};
+```
+
+See the docs for [Appearance](https://reactnative.dev/docs/appearance) and [useColorScheme](https://reactnative.dev/docs/usecolorscheme) for more information.
+
+## Moving AppleTV to react-native-tvos
+
+As part of our Lean Core effort and to bring AppleTV in line with other platforms like React Native Windows and React Native MacOS, we’ve started to remove AppleTV specific code from core.
+
+Going forward, Apple TV support for React Native will be maintained in [react-native-community/react-native-tvos](https://github.com/react-native-community/react-native-tvos) along with the corresponding `react-native-tvos` NPM package. This is a full fork of the main repository, with only the changes needed to support Apple TV.
+
+Releases of `react-native-tvos` will be based on the public release of React Native. For this 0.62 release of `react-native` please upgrade AppleTV projects to use `react-native-tvos` 0.62.
+
+## Other Improvements
+
+- **LogBox**: We’re adding the new LogBox error and warning experience as an opt-in using unstable_enableLogBox.
+- **React DevTools v4**: This change includes an upgrade to the [latest React DevTools](https://reactjs.org/blog/2019/08/15/new-react-devtools.html) which offers significant performance gains, an improved navigation experience, and full support for React Hooks.
+- **Accessibility improvements**: We’ve made improvements to accessibility including adding [accessibilityValue](https://github.com/facebook/react-native/commit/7df3eea1a79f12c2dfff1976d0cef605a83232ec), missing props on [Touchables](https://github.com/facebook/react-native/commit/8c0c860e38f57e18296f689e47dfb4a54088c260), onSlidingComplete [accessibility events](https://github.com/facebook/react-native/commit/c7aa6dc8270c0eabc913fe6c617c8131e3f4b3c5), and changing the default role of Switch component from ‘button’ to ‘swtich’
+
+## Breaking Changes
+
+- **Remove PropTypes**: We're removing propTypes from core components in order to release the app size impact of React Native core and to favor static type systems which check at compile time instead of runtime.
```

Comment:
```suggestion
- **Remove PropTypes**: We're removing `propTypes` from core components in order to release the app size impact of React Native core and to favor static type systems which check at compile time instead of runtime.
```